clients/cpp-console/src/cppCore/cppCore.vcxproj

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The cpp Core is only part of the windows build since that is where the tests are happening. It should be under the windows folder in the src dir not it's own.

Reply:
What about cppSampleTests - should this also be in the windows folder?

Reply:
yup they only run on windows right? I could see an argument for a tests folder though